lib/vsc/utils/run.py

```diff
@@ -532,13 +521,10 @@ def _loop_process_output(self, output):
 class RunAsync(Run):
     """Async process class"""
 
-    def _prep_module(self, modulepath=None, extendfromlist=None):
+    def _prep_module(self):
         # these will provide the required Popen, PIPE and STDOUT
-        if modulepath is None:
-            modulepath = PROCESS_MODULE_ASYNCPROCESS_PATH
-        if extendfromlist is None:
-            extendfromlist = ['send_all', 'recv_some']
-        super(RunAsync, self)._prep_module(modulepath=modulepath, extendfromlist=extendfromlist)
+        from vsc.utils import assyncprocess
```

Comment:
setting __path__ correctly fixes hits.

e.g. put this somewhere, before importing the test 
```
import vsc.utils
vsc.utils.__path__ = <path to vsc.utils you want to test> + vsc.utils.__path__
```